rules/windows/defense_evasion_execution_suspicious_psexesvc.toml

```
@@ -0,0 +1,39 @@
+[metadata]
+creation_date = "2020/08/14"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/08/14"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies suspicious psexec activity which is executing from the psexev service that has been renamed, possibly to evade detection."
```

```suggestion
description = "Identifies suspicious psexec activity which is executing from the psexec service that has been renamed, possibly to evade detection."
```

Should this be psexec? I can't remember what the service name is called off the top of my head.